src/main/java/io/zksync/domain/token/NFT.java

```diff
@@ -0,0 +1,30 @@
+package io.zksync.domain.token;
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
+import java.math.RoundingMode;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class NFT implements TokenId {
+    
+    private Integer id;
+
+    private String symbol;
+
+    private String creatorId;
+
+    private String contentHash;
+
+    @Override
+    public BigDecimal intoDecimal(BigInteger amount) {
+        return new BigDecimal(amount)
+            .setScale(1)
+            .divide(BigDecimal.TEN.pow(1), RoundingMode.DOWN);
```

Comment:
Not sure what this operation does

Reply:
This method converts amount of NFT (always 1) to decimals representation. As I understand ZkSync uses 1 decimals for NFT

Reply:
We should not, I have checked and we indeed use 1 for some reason. This is strange. I will talk to the team